### ISSUE TITLE
Flow control in templates

### ISSUE THREAD

**Issue by becdetat** [swxben/docx-template-engine#4]
Flow control in the template. For example:
- `{{foreach (var name in Names)}} {{name}}, {{endforeach}}`

**Comment by iconbloom** [comment]
Does this flow control can be use?

**Comment by becdetat** [comment]
@iconbloom No, it was just an idle thought that I never implemented. If you're interested in implementing it I would happy to accept a PR - I haven't needed to use this project for a long time ;-)
